### DISCUSSION TITLE
Can't seem to get it working

### DISCUSSION THREAD

**Discussion by paerbac** [timrobertsdev/hotcorners#10] [Q&A]
I installed git and rust, and I moved the exe from the debug to the releases folder I created (cause your command is refering to a non-existent folder), but I can't seem to get it working, I can run the exe, but it doesn't do anything. Hoping you can help me out

**Comment by timrobertsdev** [comment]
Hi, sorry to hear that this project isn't working for you. Nothing happens when you bring the mouse pointer to the top left corner after starting the exe? What build of Windows are you on?
